src/views/CountryProfile.js

```diff
@@ -1,12 +1,98 @@
 import React from "react";
 import Footer from "../components/footer";
+import {
+    Container
+} from "reactstrap";
+
+
+import Highcharts from "highcharts";
+import HighchartsReact from "highcharts-react-official";
+import highchartsMap from "highcharts/modules/map";
+import africaMapData from "@highcharts/map-collection/custom/africa.geo.json";
+
+var data = require('../assets/data/trial.json');
```

Comment:
Consider using **let**. You can read more [here](https://dev.to/sarah_chima/var-let-and-const--whats-the-difference-69e)